<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>35</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Select A Color</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="ColorBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="redColor">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>red</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="yellowColor">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>yellow</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="greenColor">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>green</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="blueColor">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>blue</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="purpleColor">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>purple</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pinkColor">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>pink</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ballImage">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>ballImage</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>ColorBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
